mne/viz/tests/test_misc.py

```diff
@@ -66,6 +66,12 @@ def test_plot_filter():
     iir = create_filter(data, sfreq, l_freq, h_freq, method='iir')
     plot_filter(iir, sfreq)
     plt.close('all')
+    iir = create_filter(data, sfreq, l_freq, h_freq, method='iir', iir_params={'output': 'ba'})
+    plot_filter(iir, sfreq, compensate=True)
```

Comment:
@agramfort like this?
Without my fix, this would fail, with the fix this will show correct filter plots.

Reply:
Perfect!